readme.md

```diff
@@ -79,9 +79,9 @@ interview if you aren't able to solve these questions 😁.
 
 ## 💡 Tips for a Successful Interview
 
-- We conduct our interviews on [coderpad.io](https://coderpad.io/). It's similar
```

Comment:
We might be switching to another platform in the future (i.e. HackerRank), so the proposed new wording removes mention of the exact platform we use.